reinforcement_learning/rl_mujoco_ray/rl_hopper_ray_gymEnv.ipynb

```diff
@@ -0,0 +1,534 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    #     instance_type = "ml.p2.8xlarge"
</pre><p>Remove commented-out instance type? Is there a reason why customers may want to use the 8xl instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3174/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Here is an example of a GPU instance, I will remove this line.